topics/snuba-commit-log.yaml

```diff
@@ -18,3 +18,4 @@ topic_creation_config:
   compression.type: lz4
   cleanup.policy: compact,delete
   min.compaction.lag.ms: "3600000"
+partitions: 1 # Commit log topic must always have one partition
```

Comment:
Nit: It would be nicer to see the topic name and partitions together in the top of these files since partitions are an important primitive of kafka.

Reply:
That's a fair point, decided to leave it here now though next to the topic_creation_config, as they both relate to topic creatoin